src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/build/netstandard2.0/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.targets

```diff
@@ -1,5 +1,15 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Target Name="_ResolveInputArguments">
+
+  <!--
+    Reports compatibility errors between these targets and the Razor SDK that were discovered during
+    the evaluation phase.
+  -->
+  <Target Name="_ReportRazorSdkCompatibilityError" Condition="'@(_RazorSdkCompatibilityError)'!=''" BeforeTargets="RazorResolveGenerateInputs">
+    <Error Text="%(_RazorSdkCompatibilityError.Identity)"/>
```

Comment:
This would stop after printing the first error. Just use `@(_RazorSdkCompatibilityError)` if all the scenarios are mutually exclusive.

Reply:
Cool beans 👍